docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,1981 @@
+
+Canvas::
+* Markdown element auto-applies text changes {kibana-pull}133318[#133318]
+* Lines operations keybindings {kibana-pull}132914[#132914]
+* Detailed tooltip {kibana-pull}131116[#131116]
+
+Cases::
+* Improved the cases search bar functionality. The search functionality will only consider the title and description fields {kibana-pull}136776[#136776]
+* Performance improvements were made to reduce the time required to create, update, and delete cases and comments. In our testing we saw around a half second reduction in the round trip time for the UI requests {kibana-pull}136452[#136452]
+
+Dashboard::
+Hide controls callout when the `hideAnnouncements` setting is `true` {kibana-pull}136410[#136410]
+
+Design::
+* Adds an H1 tag with the workpad title when viewing workpads {kibana-pull}135504[#135504]
+* Improve keyboard navigation in Discover top nav menu {kibana-pull}134788[#134788]
+
+Discover::
+* Improves the HTML formatting of fields with a list of values {kibana-pull}136684[#136684]
+* Adds support for accessing the edit field flyout from the document explorer column popover {kibana-pull}135277[#135277]
+* Adds support for copying the query from the add rule flyout {kibana-pull}135098[#135098]
+* Adds focus to h1 on navigate for single document and surrounding document views {kibana-pull}134942[#134942]
+* Improves the creation and editing of "Elasticsearch query" rule in Management {kibana-pull}134763[#134763]
+* Adds data view changed warning after alert rule created {kibana-pull}134674[#134674]
+* Make 'Test query' button pretty {kibana-pull}134605[#134605]
+* Improves the document explorer timestamp tooltip accessibility {kibana-pull}134411[#134411]
+* Adds focus to h1 element when client side routing is executed {kibana-pull}133846[#133846]
+* Adds an option to copy column values to Clipboard in Document Explorer {kibana-pull}132330[#132330]
+
+Elastic Security::
+For the Elastic Security 8.4.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Fleet::
+* Remove Kubernetes Package granularity {kibana-pull}136622[#136622]
+* Elastic-agent manifests: align with elastic-agent repo; add comments {kibana-pull}136394[#136394]
+* Configure source URI in global settings and in agent policy settings {kibana-pull}136263[#136263]
+* Adds Kubernetes in platforms selection list && update managed agent installation steps {kibana-pull}136109[#136109]
+* That PR will enable user to write custom ingest pipeline for Fleet installed datastream {kibana-pull}134578[#134578]
+* Update manifests for agent on kubernetes with new permissions {kibana-pull}133495[#133495]
+* Adds support for a textarea type in integrations {kibana-pull}133070[#133070]
+
+Kibana Home & Add Data::
+Adds AnalyticsNoDataPage {kibana-pull}134172[#134172]
+
+Lens & Visualizations::
+* Adds the ability to set top values limit to 10,000 in *Lens* {kibana-pull}136399[#136399]
```

Comment:
@ghudgins Include/exclude on top values is called out as feature while everything else is an "enhancement" - do you think this split makes sense or should we highlight something else instead/as well?